config/routes.rb

```diff
@@ -6,15 +6,11 @@
 
   root to: 'start_page#show'
 
-  resources :personal_details, only: %i[new create edit update], path: 'personal-details'
-  resources :contact_details, only: %i[new create edit update], path: 'contact-details'
-  resources :degrees, only: %i[new create edit update]
-  resources :qualifications, only: %i[new create edit update]
+  get 'welcome', to: 'welcome#show'
 
   get 'sign-up', to: 'magic_link/sign_up#new', as: :new_sign_up
   post 'sign-up', to: 'magic_link/sign_up#create', as: :sign_up
 
   get 'check-your-answers', to: 'check_your_answers#show'
   get 'application', to: 'tt_applications#show', as: :tt_application
```

Comment:
```suggestion
```

Reply:
ops, I missed it! I've searched for others `tt` string in the project, this is the last one.